scala/private/rule_impls.bzl

```diff
@@ -132,17 +132,33 @@ def _expand_location(ctx, flags):
 def _join_path(args, sep = ","):
   return sep.join([f.path for f in args])
 
-def _compile_scala(ctx, output, sources, cjars, all_srcjars, buildijar,
-                   transitive_compile_jars, resource_strip_prefix, resources,
-                   resource_jars, labels, in_scalacopts):
+def compile_scala(ctx,
+                  target_label,
+                  output,
+                  manifest,
+                  statsfile,
+                  sources,
+                  cjars,
+                  all_srcjars,
+                  buildijar,
+                  transitive_compile_jars,
+                  plugins,
+                  resource_strip_prefix,
+                  resources,
+                  resource_jars,
+                  labels,
+                  in_scalacopts,
+                  print_compile_time,
+                  expect_java_output,
+                  scalac_jvm_flags = []):
```

Comment:
Why does the scalac_jvn_flags have a default value?

Reply:
We have virtually never used this in practice and I just put it in. Would you prefer I remove it?

Reply:
I meant removing the default. Of course we can (and I assume will) iterate.

Reply:
I'd suggest to keep it as is (if it's alright with you) and remove the default